src/dispatchers.ts

```diff
@@ -235,18 +252,31 @@ export async function followersDispatcher(
     handle: string,
 ) {
     console.log('Followers Dispatcher');
-    const results = (await ctx.data.db.get<string[]>(['followers'])) || [];
-    console.log(results);
-    let items: Person[] = [];
-    for (const result of results) {
-        try {
-            const thing = await lookupPerson(ctx, result);
-            if (thing instanceof Person) {
-                items.push(thing);
-            }
-        } catch (err) {
-            console.log(err);
-        }
+    let items: Recipient[] = [];
+    const fullResults = await ctx.data.db.get<any[]>(['followers', 'expanded']);
+    if (fullResults) {
+        items = (await Promise.all(
+            fullResults.map((result, i): Promise<Recipient> => {
+                return {
+                    ...result,
+                    id: new URL(result.id),
+                    inboxId: new URL(result.inbox)
```

Comment:
For efficient delivery of activities:

```suggestion
                    inboxId: new URL(result.inbox),
                    endpoints: result.endpoints?.sharedInbox != null
                        ? { sharedInbox: result.endpoints.sharedInbox }
                        : null,
```